Documentation/distributions.md

```diff
@@ -7,6 +7,20 @@ The [CoreOS releases page](https://coreos.com/releases/) lists the version of rk
 
 If the version of rkt included in CoreOS is too old, it's fairly trivial to fetch the desired version [via a systemd unit](install-rkt-in-coreos.md)
 
+## Ubuntu/Debian
```

Comment:
Should this only mention Debian? AFAIK, packages from Debian-sid are not automatically copied to Ubuntu.